components/code-editor/typings.ts

```diff
@@ -3,11 +3,17 @@
  * found in the LICENSE file at https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/LICENSE
  */
 
-import { editor } from 'monaco-editor';
+import { editor, Environment } from 'monaco-editor';
```

Comment:
@yangjunhan Should not import the type declaration here. Forcing users to install monaco-editor even if they are not actually using it.